addons/rook/template/base/install.sh

```diff
@@ -988,3 +992,14 @@ function rook_prompt_migrate_from_longhorn() {
         bail "Not migrating"
     fi
 }
+
+# wait for all pods in rook-ceph namespace to transition to Running status
+function rook_maybe_wait_for_rollout() {
```

Comment:
i wonder if rather than waiting for the pods here we could wait for the conditions in the cephcluster crd?

https://rook.io/docs/rook/v1.10/CRDs/Cluster/ceph-cluster-crd/#conditions

Reply:
Also, what about we add this check under: https://github.com/replicatedhq/kURL/blob/main/addons/rook/template/base/install.sh#L344

Reply:
I think we can close this one